client/me/purchases/payment-methods/payment-method.tsx

```diff
@@ -26,6 +26,7 @@ export default function PaymentMethod( { paymentMethod }: { paymentMethod: Store
 					name={ paymentMethod.name }
 					expiry={ paymentMethod.expiry }
 					isExpired={ paymentMethod.is_expired }
+					razorpayVpa={ 'razorpay_vpa' in paymentMethod ? paymentMethod.razorpay_vpa : undefined }
```

Comment:
I think this makes sense as is, but I wonder if we could just reuse the `email` field that's already present as a prop here...  Something like:
```javascript
email={ 'razorpay_vpa' in paymentMethod ? paymentMethod.razorpay_vpa : paymentMethod.email } 
```
Anyway, the `razorpay_vpa` it's not really an email, so I'm not comfortable with this idea, but I'm just throwing it out there in case it inspires a better one 😆